app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -541,6 +557,33 @@ class Result extends React.Component {
     });
   };
 
+  nextSubmission = (direction) => () => {
+    const url = Routes.next_grouping_assignment_submission_result_path(
+      this.state.props.assignment_id, this.state.props.submission_id, this.state.props.result_id
+    );
+
+    $.ajax({
+      url: `${url}?direction=${direction}`
```

Comment:
In general prefer passing in a `data` option rather than constructing the URL string manually.